### PR TITLE
UCP: add profiling to eager tag recv

### PR DESCRIPTION
**Pull request by alex-mikheev**
@yosefe

### CODE REVIEW

File: src/ucp/tag/eager_rcv.c [hunk 1]
```diff
@@ -60,6 +60,7 @@ ucp_eager_handler(void *arg, void *data, size_t length, unsigned am_flags,
     size_t recv_len;
     ucp_tag_t recv_tag;
 
+    UCS_PROFILE_SCOPE_BEGIN();
```
Comment (yosefe) [inline src/ucp/tag/eager_rcv.c]: can we use UCS_PROFILE_FUNC without losing inline attribute?

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/5541/ for details (Mellanox internal link).

**Comment by yosefe** [comment]
bot:mlx:retest

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/5578/ for details (Mellanox internal link).

**Pull request merged**